code/frameworks/nextjs/src/nextImport/webpack.ts

```diff
@@ -0,0 +1,29 @@
+import type { Configuration as WebpackConfig } from 'webpack';
+import semver from 'semver';
+import { IgnorePlugin } from 'webpack';
+import { getNextjsVersion } from '../utils';
+
+export function configureNextImport(baseConfig: WebpackConfig) {
+  const nextJSVersion = getNextjsVersion();
+
+  const isNext12 = semver.satisfies(nextJSVersion, '~12');
+  const isNext13 = semver.satisfies(nextJSVersion, '~13');
```

Comment:
Also curious about this check regarding prerelease

Reply:
Yep, it works: https://jubianchi.github.io/semver-check/#/~13/13.0.6-canary.3